### PR TITLE
Support imdbid in search

### PR DESCRIPTION
**Pull request by morpheus133**
#### Database Migration
 NO

#### Description
Add search possibilities with imdbid in the newznab indexers if it supports them.

#### Todos
- [x] Tests   Will done if modification is acceptable
- [x] Documentation Will done if modification is acceptable

#### Issues Fixed or Closed by this PR
* #1829

### CODE REVIEW

File: src/NzbDrone.Core/Indexers/Newznab/NewznabRequestGenerator.cs [hunk 1]
```diff
@@ -57,6 +57,18 @@ private bool SupportsTvdbSearch
...
             }
         }
+        private bool SupportsImdbSearch
+        {
+            get
+            {
+                var capabilities = _capabilitiesProvider.GetCapabilities(Settings);
+
+                return capabilities.SupportedTvSearchParameters != null &&
+                       capabilities.SupportedTvSearchParameters.Contains("imdbid") &&
```
Comment (markus101) [inline src/NzbDrone.Core/Indexers/Newznab/NewznabRequestGenerator.cs]: Do indexers actually support this parameter?

Comment (morpheus133) [reply src/NzbDrone.Core/Indexers/Newznab/NewznabRequestGenerator.cs]: Yes!
At least Jackett support only this:
https://github.com/Jackett/Jackett/wiki/Definition-format

"Modes: # Specify which torznab search modes and attributes are supported by the
           # indexers.
           # Implementation note: Jackett doesn't care very much about this but you
           # still should specify the correct modes.
           # Especially don't include imdbid if the tracker doesn't support
           # searching by imdbid.
      search: [q, imdbid]
      **tv-search: [q, season, ep, imdbid]**
      movie-search: [q, imdbid]"

**Comment by morpheus133** [comment]
Than this is not for #1829. But implementation is still valid. 
For Jackett imdbid based search will be useful. 
Who can we continue with this pull request? (Did I need to create and Issue for support imdbid based search?)

**Comment by morpheus133** [comment]
@markus101 What is the situation with this request? 
Did I need to modify something?
Thx

**Comment by markus101** [comment]
No nothing else on your end. Since we're not updating v2 outside of major bugfixes I've merged your changes into `phantom-develop` in https://github.com/Sonarr/Sonarr/commit/24fba7a36dc0ff5300a02522f3d0b61bb35aca7f

**Pull request closed without merging**